tests/ossfuzz/fuzz_decode_raw_lzma2.c

```diff
@@ -0,0 +1,52 @@
+///////////////////////////////////////////////////////////////////////////////
+//
+/// \file       fuzz_decode_raw_lzma2.c
+/// \brief      Fuzz test program for liblzma lzma_raw_decoder() w/ LZMA2
+//
+//  Author:     Maksym Vatsyk
+//
+//  Based on Lasse Collin's original fuzz.c fuzzer for liblzma
+//
+//  This file has been put into the public domain.
+//  You can do whatever you want with this file.
+//
+///////////////////////////////////////////////////////////////////////////////
+
+#include <inttypes.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include "lzma.h"
+#include "fuzz_common.h"
+
+
+extern int
+LLVMFuzzerTestOneInput(const uint8_t *inbuf, size_t inbuf_size)
+{
+	// Initialize filter chain for lzma_raw_decoder() call
+	// Use single LZMA2 filter for decoding
+	lzma_options_lzma opt_lzma;
+	
+	lzma_filter filters[LZMA_FILTERS_MAX + 1];
+	filters[0].id = LZMA_FILTER_LZMA2;
+	filters[0].options = &opt_lzma;
+	filters[1].id = LZMA_VLI_UNKNOWN;
+
+	// Initialize a raw decoder using the above filter chain.
+	lzma_stream strm = LZMA_STREAM_INIT;
+	lzma_ret ret = lzma_raw_decoder(&strm, filters);
+	
+	if (ret != LZMA_OK) {
+		// This should never happen unless the system has
+		// no free memory or address space to allow the small
+		// allocations that the initialization requires.
+		fprintf(stderr, "lzma_raw_decoder() LZMA 2 failed (%d)\n", ret);
```

Comment:
Since we treat tabs at 8 character width, this line is more than 80 characters. We try to always wrap our lines at 80 characters